### PR TITLE
docs: fix antd version in codesandbox preview link

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

antd 版本改丢了，codesandbox 里打开缺少依赖。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     --      |
| 🇨🇳 Chinese |   --        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by xrkffgg — APPROVED** [review]

File: .dumi/theme/slots/Header/index.tsx [hunk 1]
```diff
@@ -358,7 +357,7 @@ const Header: React.FC = () => {
               content={menu}
               trigger="click"
               open={menuVisible}
-              arrowPointAtCenter
+              arrow={{ arrowPointAtCenter: true }}
```
Comment (MadCcc) [inline .dumi/theme/slots/Header/index.tsx:360]: 发现这个应该叫 `pointAtCenter` 的

Comment (afc163) [reply .dumi/theme/slots/Header/index.tsx:360]: 对，搞错了。我还奇怪了一下。

Comment (afc163) [reply .dumi/theme/slots/Header/index.tsx:360]: 废弃掉，搞个新的吧。

File: .dumi/theme/plugin.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ import sylvanas from 'sylvanas';
 class AntdReactTechStack extends ReactTechStack {
   // eslint-disable-next-line class-methods-use-this
   generatePreviewerProps(...[props, opts]: any) {
+    console.log(props, opts);
```
Comment (li-jia-nan) [inline .dumi/theme/plugin.ts:13]: 这个 log 是忘记删了还是故意留着的？

Comment (li-jia-nan) [reply .dumi/theme/plugin.ts:13]: 那就是不小心留下的了，我去删掉吧

**Pull request merged**